examples/Org.Website.Samples/QuerySample.cpp

```diff
@@ -64,7 +64,7 @@ void generate_users(std::shared_ptr<imap> users) {
 }
 
 int main() {
-    hazelcast_client hz;
+    hazelcast::client::hazelcast_client hz;
```

Comment:
There is `using namespace hazelcast::client` written at the top of this example.
I think we should either delete the `using namespace` statement or leave this line as it was.

There are other examples with the same thing.

Reply:
Reverted back the changes in examples.